pwa/app/components/tba/banner.tsx

```diff
@@ -0,0 +1,223 @@
+import { Award, Event } from '~/api/v3';
+import { getNormalizedName } from '~/lib/api/AwardType';
+import { cn } from '~/lib/utils';
+
+export function AwardBanner({
+  award,
+  event,
+  className,
+}: {
+  award: Award;
+  event: Event;
+  className?: string;
+}) {
+  return (
+    <Banner
+      title={getNormalizedName(award.award_type, event.event_type, event.year)}
+      description={event.name}
+      className={className}
+      year={event.year}
+    />
+  );
+}
+
+export function Banner({
+  title,
+  description,
+  year,
+  className,
+}: {
+  title?: string;
+  description?: string;
+  year?: number;
+  className?: string;
+}) {
+  const formattedTitle = (title ?? '')
+    .toUpperCase()
+    .split(/(FIRST)/) // Split the string to isolate "FIRST"
+    .map((part, index) =>
+      part === 'FIRST' ? <em key={index}>{part}</em> : part,
+    );
+
+  const isWffa = title === 'Woodie Flowers Finalist Award';
```

Comment:
Do we also need to special case WFA (from champs)?

Reply:
Alternatively maybe we should set the size based on the character length rather than the award name.